tsconfig.json

```diff
@@ -26,10 +19,6 @@
     "emitDeclarationOnly": true,
     "outDir": "dist"
   },
-  "include": [
-    "./src"
-  ],
-  "exclude": [
-    "**/*.test.ts"
-  ]
+  "include": ["./src"],
+  "exclude": ["**/*.test.ts", "**/*.test.tsx"]
```

Comment:
If we convert the hook test file to a `.ts` file, we can revert this line change.

Reply:
Reverted